server/controllers/SeriesController.js

```diff
@@ -41,7 +41,7 @@ class SeriesController {
     const hasUpdated = req.series.update(req.body)
     if (hasUpdated) {
       await this.db.updateEntity('series', req.series)
-      SocketAuthority.emitter('series_updated', req.series)
+      SocketAuthority.emitter('series_updated', req.series.toJSON())
     }
     res.json(req.series)
```

Comment:
Should `toJSON()` be added for the API response?

```suggestion
    res.json(req.series.toJSON())
```

Reply:
It's not required since the socket library is going to make that call but in our case I think it is a good idea because we differentiate between toJSONExpanded, etc.